htdocs/core/class/html.form.class.php

```diff
@@ -1153,14 +1153,14 @@ public function select_thirdparty_list($selected = '', $htmlname = 'socid', $fil
 		}
 
 		$sql.= " FROM ".MAIN_DB_PREFIX ."societe as s";
-		if (!$user->rights->societe->client->voir && !$user->socid) $sql .= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
+		if (!$user->rights->societe->client->voir && !$user->socid) $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe_commerciaux as sc ON s.rowid = sc.fk_soc";
```

Comment:
Because there is a condition later on a field of sc tabke, using a left join here should be same than using a strict inner join, except it is slower.